python/cudf/cudf/utils/applyutils.py

```diff
@@ -105,14 +105,15 @@ def apply_chunks(
     return applychunks.run(df, chunks=chunks, tpb=tpb)
 
 
+@acquire_spill_lock()
 def make_aggregate_nullmask(df, columns=None, op="__and__"):
 
     out_mask = None
     for k in columns or df._data:
         col = cudf.core.dataframe.extract_col(df, k)
         if not col.nullable:
             continue
-        nullmask = df[k].nullmask
+        nullmask = cudf.Series(df[k]._column.nullmask)
```

Comment:
Can this use `from_data`?

Reply:
We can just use `as_column` itself, changed it.